src/pages/login.tsx

```diff
@@ -1,77 +1,133 @@
-      <form
-        data-testid={isSignup ? 'login-signup-form' : 'login-signin-form'}
-        onSubmit={handleSubmit}
-        className="flex max-w-md flex-col gap-4 "
-      >
-        <h2 className="mb-1 text-base font-semibold">
-          {isSignup ? 'Sign Up' : 'Sign In'}
-        </h2>
-        <div>
-          <div className="mb-2 block">
-            <Label htmlFor="email1">Your email</Label>
-          </div>
-          <Input
-            id="email"
-            data-testid="login-email"
-            type="email"
-            placeholder="jane.doe@example.com"
-            required
-          />
-        </div>
-        <div>
-          <div className="mb-2 block">
-            <Label htmlFor="password1">Your password</Label>
-          </div>
-          <Input
-            data-testid="login-password"
-            id="password1"
-            type="password"
-            required
+    <div className="flex flex-col justify-center relative h-full w-1/4 overflow-y-auto m-10 pb:12 mx-auto">
+      {props?.providers?.['credentials'] && isSignup ? (
+        <SignUpForm onSubmit={handleSignUp} />
+      ) : (
+        <SignInForm onSubmit={handleSignIn} />
+      )}
+      {error && (
+        <DangerAlert className="mt-3" data-testid="login-error">
+          <span className="font-medium">Auth Error!</span> {parsedError()}
+        </DangerAlert>
+      )}
+      {props?.providers?.google && (
+        <Button
+          onClick={() => signIn('google', { redirect: true, callbackUrl: '/' })}
+          variant="outline"
+          className="mt-4"
+        >
+          <Image
+            height="24"
+            width="24"
+            alt="Google Provider"
+            id="provider-logo"
+            src="https://authjs.dev/img/providers/google.svg"
           />
-        </div>
-
-        {isSignup && (
-          <div>
-            <div className="mb-2 block">
-              <Label htmlFor="password2">Confirm password</Label>
-            </div>
-            <Input
-              id="password2"
-              data-testid="login-confirm-pswd"
-              type="password"
-              required
-            />
-          </div>
-        )}
-        <Button data-testid="login-submit-btn" type="submit">
-          Submit
+          <span>Sign in with Google</span>
         </Button>
-      </form>
+      )}
+      <p className="mt-4 dark:text-white">
+        {isSignup ? (
+          <>
+            Already have an account?{' '}
+            <Link href="/login?signup=0">
+              <span className="text-blue-600 hover:underline">Sign in</span>
+            </Link>
+          </>
+        ) : (
+          <>
+            {' '}
+            Don&apos;have an account?{' '}
+            <Link href="/login?signup=1">
+              <span className="text-blue-600 hover:underline">Sign up</span>
+            </Link>
+          </>
+        )}
+      </p>
```

Comment:
I hightly suggest  you to divide sign up and sign in to different pages. Not only the  endpoints  for auth handling, but the pages also. Reading the component itself is super hard and is huge also. Other  than that,  everything  is looking good!

Reply:
I couldn't reproduce the issue: 

<img width="685" alt="image" src="https://github.com/AlanMamphs/graphiql-app/assets/14315501/c999e3fc-7968-4a33-a532-6c8e75721e5a">

Reply:
That's odd. I don't event checkout your PR, previous  screenshot is from vercel deployment  as well, as next one. 

I would suggest you to check UI response from different browsers. I am using Firefox Developer Edition,

![image](https://github.com/AlanMamphs/graphiql-app/assets/65208028/46acb365-50e4-4785-9c3a-1f4d18a50d76)

Reply:
It is fine in my chrome and firefox. It must be something on your end. Let's fix it later